src/components/MultiHopTrade/hooks/useGetTradeQuotes/getTradeQuoteInput.ts

```diff
@@ -70,41 +67,75 @@ export const getTradeQuoteInput = async ({
 
   switch (chainNamespace) {
     case CHAIN_NAMESPACE.Evm: {
-      const supportsEIP1559 = supportsETH(wallet) && (await wallet.ethSupportsEIP1559())
+      const supportsEIP1559 =
+        hasWallet && wallet && supportsETH(wallet) && (await wallet.ethSupportsEIP1559())
```

Comment:
nitpick
```suggestion
        supportsETH(wallet) && (await wallet.ethSupportsEIP1559())
```

Reply:
0de700b10c - kept in other places since relevant to current types